analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ExecutingSqlQueries.Net46.MonoSqlLite.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using Mono.Data.Sqlite;
+
+namespace Test
+{
+    public class C
```

Comment:
Maybe rename the class to `Sample` for consistency with other tests. `C` looks pretty arbitrary and confusing.